packages/sources/coingecko-test/src/endpoint/crypto.ts

```diff
@@ -1,23 +1,23 @@
 import { PriceEndpoint } from '@chainlink/external-adapter-framework/adapter'
-import { BatchWarmingTransport } from '@chainlink/external-adapter-framework/transports/batch-warming'
 import {
   buildBatchedRequestBody,
   constructEntry,
   CryptoEndpointTypes,
   cryptoInputParams,
 } from '../crypto-utils'
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
 
-const batchEndpointTransport = new BatchWarmingTransport<CryptoEndpointTypes>({
-  prepareRequest: (params, config) => buildBatchedRequestBody(params, config),
+const transport = new HttpTransport<CryptoEndpointTypes>({
+  prepareRequests: (params, config) => buildBatchedRequestBody(params, config),
   parseResponse: (params, res) =>
     params.map((requestPayload) =>
-      constructEntry(res, requestPayload, requestPayload.quote.toLowerCase()),
+      constructEntry(res.data, requestPayload, requestPayload.quote.toLowerCase()),
     ),
 })
 
 export const endpoint = new PriceEndpoint<CryptoEndpointTypes>({
   name: 'crypto',
-  aliases: ['crypto-batched', 'batched', 'batch'],
-  transport: batchEndpointTransport,
+  aliases: ['crypto-batched', 'batched', 'batch', 'price'],
```

Comment:
I think we can use the new `CryptoPriceEndpoint` Alejo added here. Should be available in the latest version.

Reply:
Updated!